libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -554,7 +558,7 @@ interface BlockingBreezServices {
    sequence<Payment> list_payments(ListPaymentsRequest request);
 
    [Throws=SdkError]
-   void sweep(string to_address, u64 fee_rate_sats_per_vbyte);
+   SweepResponse sweep(string to_address, u64 fee_rate_sats_per_vbyte);
```

Comment:
Perhaps also use SweepRequest as argument that wraps all these arguments. I know it is not part of this PR but since we are changing the interface here anyway it might worth doing it in one shot.